translations/ja.json

```diff
@@ -1,54 +1,54 @@
-	"A firewall is likely blocking minikube from reaching the internet. You may need to configure minikube to use a proxy.": "ファイアウォールによって、minikube がインターネットに繋がることができてない可能性があります。minikube がプロキシーを使うように設定する必要があるかもしれません",
-	"A set of apiserver IP Addresses which are used in the generated certificate for kubernetes.  This can be used if you want to make the apiserver available from outside the machine": "Kubernetes 用に生成された証明書で使用されている一連の APIサーバーの IP アドレスのセット。 マシンの外部から API サーバーを利用できるようにする場合に使用します",
+	"\"{{.name}}\" profile does not exist, trying anyways.": "「{{.name}}」プロファイルは存在しませんが、続行します。",
+	"'none' driver does not support 'minikube docker-env' command": "'none' ドライバーは 'minikube docker-env' コマンドをサポートしていません",
+	"'none' driver does not support 'minikube mount' command": "'none' ドライバーは 'minikube mount' コマンドをサポートしていません",
+	"'none' driver does not support 'minikube podman-env' command": "'none' ドライバーは 'minikube podman-env' コマンドをサポートしていません",
+	"'none' driver does not support 'minikube ssh' command": "'none' ドライバーは 'minikube ssh' コマンドをサポートしていません",
+	"'none' driver does not support 'minikube ssh-host' command": "'none' ドライバーは 'minikube ssh-host' コマンドをサポートしていません",
+	"'{{.driver}}' driver reported a issue that could affect the performance.": "'{{.driver}}' ドライバーがパフォーマンスに影響しうる問題を報告しました。",
+	"'{{.driver}}' driver reported an issue: {{.error}}": "'{{.driver}}' ドライバーがエラーを報告しました: {{.error}}",
+	"- Delete and recreate minikube cluster\n\t\tminikube delete\n\t\tminikube start --driver={{.driver_name}}": "- minikube クラスターの削除と再作成をします\n\t\tminikube delete\n\t\tminikube start --driver={{.driver_name}}",
+	"- Docs https://docs.docker.com/docker-for-mac/#resources": "- ドキュメント https://docs.docker.com/docker-for-mac/#resources",
+	"- Docs https://docs.docker.com/docker-for-windows/#resources": "- ドキュメント https://docs.docker.com/docker-for-windows/#resources",
+	"- Ensure your {{.driver_name}} daemon has access to enough CPU/memory resources.": "- {{.driver_name}} デーモンが十分な CPU/メモリーリソースにアクセスできることを確認してください。",
+	"- Prune unused {{.driver_name}} images, volumes, networks and abandoned containers.\n\n\t\t\t\t{{.driver_name}} system prune --volumes": "- 未使用の {{.driver_name}} イメージ、ボリューム、ネットワーク、使われていないコンテナーを削除します。\n\n\t\t\t\t{{.driver_name}} system prune --volumes",
+	"- Restart your {{.driver_name}} service": "{{.driver_name}} サービスを再起動します",
+	"- {{.logPath}}": "- {{.logPath}}",
+	"--kvm-numa-count range is 1-8": "--kvm-numa-count の範囲は 1-8 です",
+	"--network flag is only valid with the docker/podman and KVM drivers, it will be ignored": "--network フラグは、docker/podman および KVM ドライバーでのみ有効であるため、無視されます",
+	"127.0.0.1": "127.0.0.1",
+	"\u003ctarget file absolute path\u003e must be an absolute Path. Relative Path is not allowed (example: \"/home/docker/copied.txt\")": "\u003ctarget file absolute path\u003e は絶対パスでなければなりません。相対パスは使用できません (例:「/home/docker/copied.txt」)",
+	"==\u003e Audit \u003c==": "==\u003e Audit \u003c==",
+	"==\u003e Last Start \u003c==": "==\u003e Last Start \u003c==",
```

Comment:
I didn't translate because "Audit" and "Last Start" are names of log types and in many cases, it seems there are written without translation.
If you have any other opinions, please comment.